tests/functions.test.js

```diff
@@ -17,35 +17,45 @@ describe('functions', () => {
   };
 
   describe('addNums', () => {
-    const addNums = cases.addNums;
     it('should be a function', () => {
+      const addNums = cases.addNums;
       expect(addNums).to.be.a('function');
     });
-    // write a test to check if 'addNums' returns the expected value
-    // i.e. if is called addNums(1, 2); the return value should be 3.
+    it('should add up two numbers passed to it', () => {
+      const result = cases.addNums(1, 2);
+      expect(result).to.equal(3);
```

Comment:
Since you declared `addNums` above, you can just do `expect(addNums(1, 2)).to.equal(3)`